control/plugin_manager.go

```diff
@@ -624,3 +638,64 @@ func (p *pluginManager) all() map[string]*loadedPlugin {
 	defer p.loadedPlugins.RUnlock()
 	return p.loadedPlugins.table
 }
+
+func hasPrefix(ns1 []string, ns2 []string) bool {
```

Comment:
Can this be replaced with [strings.HasPrefix](https://golang.org/pkg/strings/#HasPrefix)?